src/wh_client_cryptocb.c

```diff
@@ -66,6 +66,96 @@ int wolfHSM_CryptoCb(int devId, wc_CryptoInfo* info, void* inCtx)
         packet->pkAnyReq.type = info->pk.type;
         switch (info->pk.type)
         {
+        case WC_PK_TYPE_RSA_KEYGEN:
+            /* set size */
+            packet->pkRsakgReq.size = info->pk.rsakg.size;
+            /* set e */
+            packet->pkRsakgReq.e = info->pk.rsakg.e;
+            /* write request */
+            ret = wh_Client_SendRequest(ctx, group,
+                WC_ALGO_TYPE_PK,
+                WOLFHSM_PACKET_STUB_SIZE + sizeof(packet->pkRsakgReq),
+                rawPacket);
+            if (ret == 0) {
+                do {
+                    ret = wh_Client_RecvResponse(ctx, &group, &action, &dataSz,
+                        rawPacket);
+                } while (ret == WH_ERROR_NOTREADY);
+            }
+            if (ret == 0) {
+                if (packet->rc != 0)
+                    ret = packet->rc;
+                else {
+                    info->pk.rsakg.key->devCtx =
+                        (void*)0 + packet->pkRsakgRes.keyId;
```

Comment:
void* arithmetic is generally frowned upon.  Can you switch this to typecast to uintptr_t and then to void*?  That is the best way to keep analyzers happy and then intent of putting an integer into a pointer member.